### DISCUSSION TITLE
Adding a description field

### DISCUSSION THREAD

**Discussion by jp26jp** [nhn/tui.calendar#857] [Q&A]
I can see that `Schedule` has a `data.memo` attribute, but how can I get those details to appear in the popup template?

**Comment by adhrinae** [comment]
@jp26jp Did you mean the `raw` attribute? It's an arbitrary property that users can handle by themselves.
What you saw was [just a kind of example.](https://github.com/nhn/tui.calendar/blob/ac65d491c21b99ba18e179664c96089bd51216c7/examples/js/data/schedules.js#L136-L141)
If you want to use the `raw` property on the template, I think the `popupDetailBody` template might be useful. ([Check the Javascript tab in this page](https://nhn.github.io/tui.calendar/latest/tutorial-example16-template-for-popup)) or You need to implement your own popup and use it through [the `clickSchedule` event](https://nhn.github.io/tui.calendar/latest/Calendar#event-clickSchedule).
